routes/web.php

```diff
@@ -1080,4 +1080,6 @@ function (): void {
         ->where('application', '[0-9]+')
         ->middleware('can:view,application')
         ->name('application.experience.index');
+    Route::get('classifications', 'ClassificationsController@getClassifications');
+        //->middleware('can:view,application');
```

Comment:
I don't think we necessarily need middleware here, as most of our other lookup data is open. I'm also thinking we might want to leave this under the `v1` group and put it around line 858 just for consistency with the other lookup endpoints
```php
Route::get('classifications', 'Api\ClassificationController@index');
```